README.md

```diff
@@ -0,0 +1,183 @@
+- *[Display home page]*
+- *[Display about page]*
+- *[Display footer]*
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+## :rocket: Live Demo <a name="live-demo"></a>
+- [Live Demo Link](https://isume295.github.io/capstone-1/)
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+## 💻 Getting Started <a name="getting-started"></a>
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+In order to run this project you need:
+- GitHub Account.
+- Git bash.
+- Visual Studio Code.
+- Node JS.
+
+### Setup
+Clone this repository to your desired folder:
+Use git clone command or download ZIP folder
+sh
+  cd folder Name 
+git@github.com:isume295/capstone-1.git
+
+### Install
+Install this project with:
+npm
+sh
+  cd project Name
+  npm init -y
+
+### Usage
+To run the project, execute the following command:
+npm start or live server
+
+sh
+  GitHub Pages Server
+
+### Run tests
+To run tests, run the following command:
+ 
+ to test validation errors
+ npx hint . 
+ 
+ to test linter Errors
+ npx stylelint "**/*.{css,scss}"
+
+### Deployment
+You can deploy this project using:
+GitHub Pages
+
+https://isume295.github.io/capstone-1/
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHORS -->
+## :busts_in_silhouette: Authors <a name="authors"></a>
+
+:bust_in_silhouette: *Sumeya Ibrahim*
+
+- GitHub: [@isume295](https://github.com/isume295/)
+- Twitter: [@isume295](https://twitter.com/isume295)
+- LinkedIn: [@isume295](https://www.linkedin.com/in/sumeya-ibrahim-109002232)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FUTURE FEATURES -->
+## :telescope: Future Features <a name="future-features"></a>
+
+- [ ] *[Animations]*
+- [ ] *[registration-section]*
+- [ ] *[video Section]*
+ 
+
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- CONTRIBUTING -->
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](https://github.com/isume295/capstone-1/issues).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- SUPPORT -->
+
+## ⭐️ Show your support <a name="support"></a>
+
+If you like this project I would like to inform you that this project will improve and add other exciting features in the next release. stay tuned!!!
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- ACKNOWLEDGEMENTS -->
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+Design by Cindy Shin
+Behance: [Design](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- walkthrough -->
+
+##  Walkthrough <a name="walkthrough"></a>
+
+Feel free to check the [walkthrough video page](https://www.loom.com/share/3a1ed8cf22274d2f8a8cc7d0b886a679).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
```

Comment:
[OPTIONAL] You can move this walkthrough presentation video to the live demo section as it counts as a demo of your project